Public/Sdk/Public/Transformers/Transformer.Execute.dsc

```diff
@@ -83,10 +83,13 @@ namespace Transformer {
 
         /** Regex that would be used to extract warnings from the output. */
         warningRegex?: string;
-        
+
         /** Regex that would be used to extract errors from the output. */
         errorRegex?: string;
 
+        /** Options used to construct error regex (see the RegexOptions C# enum for valid values) */
+        errorRegexOptions?: number;

```

Comment:
>errorRegexOptions?: number; [](start = 8, length = 27)

While extensible, it feels a bit funny to expose it this way. I think multiline is the only thing we want to control here. SHould we just expose a bool?

Reply:
I can change it to bool.

Reply:
Changed to bool